Tests/KlaviyoCoreTests/__Snapshots__/EncodableTests/testEventPayloadWithoutMetadata.1.json

```diff
@@ -21,23 +21,11 @@
         }
       },
       "properties" : {
-        "App Build" : "1",
```

Comment:
Expected yes?

Reply:
Replaced this with a test called `testEventPayload` since we are including metadata when creating the event request object and also added sample properties that got removed back and now the JSON looks like, 

```json

{
  "data" : {
    "attributes" : {
      "metric" : {
        "data" : {
          "attributes" : {
            "name" : "test"
          },
          "type" : "metric"
        }
      },
      "profile" : {
        "data" : {
          "attributes" : {
            "anonymous_id" : "anon-id",
            "properties" : {

            }
          },
          "type" : "profile"
        }
      },
      "properties" : {
        "App Build" : "1",
        "App ID" : "com.klaviyo.fooapp",
        "App Name" : "FooApp",
        "App Version" : "1.2.3",
        "blob" : "blob",
        "Device ID" : "fe-fi-fo-fum",
        "Device Manufacturer" : "Orange",
        "Device Model" : "jPhone 1,1",
        "hello" : {
          "sub" : "dict"
        },
        "OS Name" : "iOS",
        "OS Version" : "1.1.1",
        "Push Token" : "",
        "SDK Name" : "swift",
        "SDK Version" : "3.2.0",
        "stuff" : 2
      },
      "time" : "2009-02-13T23:31:30Z",
      "unique_id" : "00000000-0000-0000-0000-000000000001"
    },
    "type" : "event"
  }
}

```

Reply:
Thanks for calling these out!